drift-client/src/main/java/io/airlift/drift/client/ExceptionClassification.java

```diff
@@ -78,4 +88,79 @@ public String toString()
                 .add("hostStatus", hostStatus)
                 .toString();
     }
+
+    public static Collector<ExceptionClassification, ?, ExceptionClassification> mergeExceptionClassifications()
```

Comment:
Can you use `Collectors.reducing()` of creating a custom collector?

Reply:
This is a reducer has an accumulator that means we only get one extra objected created.